### PR TITLE
Re-organize internet-required layers

### LINKED ISSUE #727: Re-organize "Internet required" layers
Consider subgroups or some other way to indicate internet-required layers in the layers panel.

Some (e.g., the [geological map from geus](https://github.com/nsidc/qgreenland/pull/725)) might be best placed alongside layers that are in the "Geology" group. Maybe we could have "Internet-required" subgroups under appropriate discipline-specific groups? E.g.,

* "Geology/Internet-required/..."
* "Basemaps/Internet-required/..."

### PR DESCRIPTION
**Pull request by MattF-NSIDC**
## Description

Closes #727 

## Checklist

If an item on this list is done _or not needed_, check it with `[x]` or click the
checkbox.

- [x] The PR description links to issues that it resolves with `closes #{issue_number}`
- [x] Config lockfile updated (`inv config.export > qgreenland/config/cfg-lock.json`)
- [x] Environment lockfile updated if needed (`conda-lock`)
- [x] Version bumped if needed (`bumpversion (major|minor|patch|prerelease|build)`)
- [x] CHANGELOG.md updated (for user-facing changes)
- [x] Documentation updated if needed
- [x] New unit tests if needed

### CODE REVIEW

File: qgreenland/config/layers/Basemaps/Internet required/online.py
Comment (trey-stafford) [inline qgreenland/config/layers/Basemaps/Internet required/online.py:6]:
```suggestion
    title="Blue Marble shaded relief and bathymetry (500m)",
```

Just noticed we used capitalization unnecessarily here. Accepting this change will trigger the need for a config lock though...

**Review by trey-stafford — APPROVED** [review]
One suggested change unrelated to your PR.

**Comment by trey-stafford** [comment]
Resolves https://github.com/nsidc/qgreenland/issues/510

File: qgreenland/config/layers/Basemaps/Internet required/online.py
Comment (MattF-NSIDC) [reply qgreenland/config/layers/Basemaps/Internet required/online.py]: ba48aeab

**Comment by MattF-NSIDC** [comment]
Waiting for more from @twilamoon-science in #727 before merge

**Comment by MattF-NSIDC** [comment]
@trey-stafford applied changes from discussion with Twila in #770

**Review by trey-stafford — APPROVED** [review]

**Pull request merged**
